azure-pipelines-cloud-integration.yml

```diff
@@ -30,21 +30,20 @@ stages:
               python.version: '3.9'
         steps:
           - bash: |
-              TAG="greatexpectations:py$(python.version)" 
+              TAG="greatexpectations:py$(python.version)"
               BUILD="docker buildx build -f docker/Dockerfile.tests --tag ${TAG} "
               BUILD+="--target test --build-arg PYTHON_VERSION=$(python.version) ."
               echo ${BUILD}
               eval ${BUILD}
 
-              PYTEST="docker run -e GE_CLOUD_ORGANIZATION_ID=$(GE_CLOUD_ORGANIZATION_ID) "
-              PYTEST+="-e GE_CLOUD_ACCESS_TOKEN=$(GE_CLOUD_ACCESS_TOKEN) -e GE_CLOUD_BASE_URL=$(GE_CLOUD_BASE_URL) "
+              PYTEST="docker run -e GX_CLOUD_ORGANIZATION_ID=$(GE_CLOUD_ORGANIZATION_ID) "
+              PYTEST+="-e GX_CLOUD_ACCESS_TOKEN=$(GE_CLOUD_ACCESS_TOKEN) -e GX_CLOUD_BASE_URL=$(GE_CLOUD_BASE_URL) "
               PYTEST+="$TAG "
               PYTEST+="pytest --random-order --cloud -m \"cloud and e2e\""
               echo ${PYTEST}
               eval ${PYTEST}
             env:
-              GE_CLOUD_BASE_URL: $(GE_CLOUD_BASE_URL)
-              GE_CLOUD_ACCESS_TOKEN: $(GE_CLOUD_ACCESS_TOKEN)
-              GE_CLOUD_ORGANIZATION_ID: $(GE_CLOUD_ORGANIZATION_ID)
+              GX_CLOUD_BASE_URL: $(GE_CLOUD_BASE_URL)
+              GX_CLOUD_ACCESS_TOKEN: $(GE_CLOUD_ACCESS_TOKEN)
+              GX_CLOUD_ORGANIZATION_ID: $(GE_CLOUD_ORGANIZATION_ID)
```

Comment:
We currently have `GE_...` saved in the Azure UI - we can change this in a subsequent PR